proposals/lip-0059.md

```diff
@@ -41,16 +41,16 @@ lastHeightOfRound(h + 2 * ROUND_LENGTH) <= maxHeightCertified.
 ```
 
 Here
-- `ROUND_LENGTH` is the round length in blockchains using the [DPoS module LIP][research:dpos-module],
+- `ROUND_LENGTH` is the round length in blockchains using the [DPoS module LIP][lip-0057],
 - `lastHeightRound()` returns the last height of the round containing the given height,
 - `maxHeightCertified` is the maximum height of a block for which the chain contains a certificate.
 
 ## Specification
 
 ### Notation
 
-- We use the constant `ROUND_LENGTH` defined in the [LIP "Define state and state transitions of DPoS module"][research:dpos-module] to denote the round length in the DPoS chain, i.e., the number of active plus randomly selected standby delegates.
-- We write `bftModule.fct` for a call to the function `fct` defined in the [BFT module][research:bft-module].
+- We use the constant `ROUND_LENGTH` defined in [LIP 0057][lip-0057] to denote the round length in the DPoS chain, i.e., the number of active plus randomly selected standby delegates.
+- We write `bftModule.fct` for a call to the function `fct` defined in the [BFT module][lip-0058].
```

Comment:
We should not refer to it as a module, right?

Reply:
Fixed, I removed this bullet point.